Makefile

```
@@ -466,6 +466,11 @@ check-unused-components:
 
 documentation: check-unused-components generate-public-docs verify-api-docs
 
+show-local-api-docs:
+	open docs/index.html
+
+doc-build: documentation show-local-api-docs
```

checkmake (0.3.2)

[warning] Target "doc-build" should be declared PHONY.

(phonydeclared)